domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -512,6 +524,7 @@ constructor(
                         }
                     }
                 }
+        recording?.mute(currentSettings.value!!.audioMuted)
```

Comment:
I just wonder, does force unwrap `currentSettings.value!!` fine to use?

Reply:
Good point.

It should be safe to unwrap at this point as the currentSettings should have been initialized, i think it'd still be a good idea to use a safe call operator regardless.

thanks!